### PR TITLE
Byol existing ocp

### PR DESCRIPTION
**Pull request by amitmangalvedkar**
Code drop checks for CIDR range check and for using `SecretManager`

### CODE REVIEW

File: aws/master-cft/byol-existing-ocp/cft-mas-core-dev.json [hunk 1]
```diff
@@ -19,7 +19,12 @@
         "BootnodeSGIngressCidrIp": {
             "Description": "Use CIDR notation to enter the IP addresses that can access the bootnode. For example, 192.12.33.3/32. To allow access from all IP addresses, enter 0.0.0.0/0.",
             "Type": "String",
-            "MinLength": "1"
+            "MinLength": "9",
+            "MaxLength": "18",
+            "Default": "0.0.0.0/0",
```
Comment (santoshjpawar) [inline aws/master-cft/byol-existing-ocp/cft-mas-core-dev.json:24]: This field should be empty as per the AWS recommendation. 
User should restrict the bootnode access to specific IP or range. So in that case, the minlen 9 will not work. Minlen 1 is to make sure user specify some value and CFT UI blocks the deployment if empty. So we can keep it as is or have minlen as 7 considering all parts of the IP can be single digit (9.8.2.1).

Comment (amitmangalvedkar) [reply aws/master-cft/byol-existing-ocp/cft-mas-core-dev.json:24]: hmm what about the decimal number post the slash _(and the slash)_ - that would make it 9.

Comment (santoshjpawar) [reply aws/master-cft/byol-existing-ocp/cft-mas-core-dev.json:24]: Yes, that's right. The CIDR format is required. Just remove the default value and it should be good to merge.

**Review by santoshjpawar — APPROVED** [review]
Looks good

**Pull request merged**